application_sdk/outputs/parquet.py

```diff
@@ -166,7 +228,7 @@ async def write_daft_dataframe(self, dataframe: "daft.DataFrame"):  # noqa: F821
             )
 
             # Upload the file to object store
-            await self.upload_file(self.output_path)
+            await self.upload_file(file_path)
```

Comment:
this might break the normal extraction flow I believe, correct me - earlier this would upload a directory `self.output_path`, now it would upload a file_path.